### ISSUE TITLE
Organize sequential information into lists

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#561]
Issue group: API
Visibility: 2
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: 

### Description
On the API integrations page, the message log places each message in sibling <div>s. The message log should be a list with each message in a list item <li>. 

![](https://dl.airtable.com/.attachments/96171a7ecf9a114ff8caf0fa8bb4d111/82f81d1a/Capturedecranle2022-03-15a09_56_19.png)

**Comment by YedidaZalik** [comment]
@amazingphilippe worked on yesterday and running tests and will create PR

**Comment by adriannelee** [comment]
@amazingphilippe To review comments.

**Comment by amazingphilippe** [comment]
Merged and waiting for release, QA

**Comment by adriannelee** [comment]
Done QA by Andrew
